README.md

```diff
@@ -130,6 +130,31 @@ To dump VCDs (into the test_run_dir subfolder) using sbt:
 testOnly chiseltest.tests.BasicTest -- -DwriteVcd=1
 ```
 
+## Verilator Backend
+
+To run your tests using the [verilator](https://www.veripool.org/wiki/verilator) simulator:
+
+1. Ensure that you have a _supported version_ of verilator installed.
+   You can [consult the CI script](https://github.com/ucb-bar/chisel-testers2/blob/master/.github/workflows/test.yml#L38)
```

Comment:
Is it worth adding how to set up a supported version of Verilator? This could be linking to the Chisel readme if it's there.

Also, instead of linking the CI scripts, perhaps just link the Chisel readme section with the version, since that's more readable and perhaps more authoritative?

Reply:
I was afraid of having two sources of truth, but I think you are correct wrt. copying it into the readme being much more readable.

Reply:
Yeah, understand the two sources of truth; I'm fine with the version not being duplicated in this document, but I think a link to the Chisel readme section containing the version would be preferable to a link to a CI script, even if the versions are pretty obvious in the CI script. Bonus: Chisel readme actually has setup directions.